pkg/auth/config/source/source.go

```diff
@@ -0,0 +1,13 @@
+package source
+
+import (
+	"context"
+	"github.com/skygeario/skygear-server/pkg/auth/config"
+	"net/http"
+)
+
+type Source interface {
+	Start() error
+	Shutdown() error
```

Comment:
Use the term Open and Close instead? Or can you clarify the intended lifecycle of Source?

Reply:
This is intended for watching configuration files (start/shutdown the watcher) later on. I do agree 'opening configuration source' sounds more natural. I'll change it to Open/Close.